core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/docker/DockerContainer.scala

```diff
@@ -101,6 +102,7 @@ object DockerContainer {
       dnsSearch.flatMap(d => Seq("--dns-search", d)) ++
       dnsOptions.flatMap(d => Seq(dnsOptString, d)) ++
       name.map(n => Seq("--name", n)).getOrElse(Seq.empty) ++
+      cpuLimit.map(c => Seq("--cpus", c.toString)).getOrElse(Seq.empty) ++
```

Comment:
i wasn't aware that you could do both `--cpus` and `--cpu-shares` on the docker run. how does this look in practice with how it behaves setting both?

Reply:
`--cpu-shares` still provides the weight/priority to cpu cycles for the container, `--cpus` just provides the cap.